addons/mail/static/src/discuss/core/public/welcome_page.js

```diff
@@ -34,6 +40,16 @@ export class WelcomePage extends Component {
         if (this.store.guest) {
             await this.personaService.updateGuestName(this.store.self, this.state.userName.trim());
         }
+        if (!this.state.audioStream) {
+            browser.localStorage.setItem("discuss_call_preview_join_mute", "true");
+        } else {
+            browser.localStorage.removeItem("discuss_call_preview_join_mute");
+        }
+        if (this.state.videoStream) {
+            browser.localStorage.setItem("discuss_call_preview_join_video", "true");
+        } else {
+            browser.localStorage.removeItem("discuss_call_preview_join_video");
+        }
```

Comment:
I would keep it as before (single line) since there is only one place where we use it, we're better parsing it in this place instead.